core/templates/pages/topic-viewer-page/navbar-breadcrumb/topic-viewer-navbar-breadcrumb.component.spec.ts

```diff
@@ -21,7 +21,7 @@ import { HttpClientTestingModule } from '@angular/common/http/testing';
 import { UrlService } from 'services/contextual/url.service';
 import { TopicViewerBackendApiService } from
   'domain/topic_viewer/topic-viewer-backend-api.service';
-import { ReadOnlyTopicObjectFactory } from
+import { ReadOnlyTopicBackendDict, ReadOnlyTopicObjectFactory } from
   'domain/topic_viewer/read-only-topic-object.factory';
 import { TopicViewerNavbarBreadcrumbComponent } from
   // eslint-disable-next-line max-len
```

Comment:
You can remove these comments

Reply:
// eslint-disable-next-line max-len ?? Next line is actually too long.

Reply:
this isn't applicable in the case for imports, you can remove that

Reply:
Done